.gitignore

```diff
@@ -10,4 +10,7 @@ tests/_output/*
 tests/_output/*
 tests/_output/*
 tests/_output/*
-tests/_output/*
\ No newline at end of file
+tests/_output/*
+
+web/assets
+config/local.php
```

Comment:
`config/local.php` need to be stored in git, and your suggest is right, `web/assets` should be removed, it will be created by `yii run/setup`